README.md

```diff
@@ -193,9 +193,9 @@ The following CMake options are available for adding/removing project features.
 </tr>
 </table>
 
-- ``Samples``: Whenever UNIT_TESTING is ON, samples are built using the default PAL (see [running samples section](#running-samples)). This means that running samples would throw errors like:
+- ``Samples``: Storage Samples are built by default using the default PAL and HTTP adapter (see [running samples](#running-samples)). This means that running samples without building an HTTP transport adapter would throw errors like:

```

Comment:
We should have a `BUILD_SAMPLES` option to simplify this.

Reply:
Oh, we do `AZ_SDK_C_NO_SAMPLES`.

Are storage samples being built even when `AZ_SDK_C_NO_SAMPLES` is on?

Reply:
`AZ_SDK_C_NO_SAMPLES` disables all samples generation.